app/info.md

```diff
@@ -0,0 +1,56 @@
+### Model Description
+
+## Summary
+
+The principle of the estimation is:
+
+1. Augment the number of admissions using the reporting. This is currently done by
+
+ n<sub>aug</sub> = n<sub>reported</sub> / % reported
+ 
+2. Use a log-linear model, parametrised through the doubling time, to simulate epidemic trajectories; see for instance Jombart et al 2020; this is implemented by the RECON package projections.
+
+3. For each admission, simulate duration of hospitalisation from provided distribution.
+
+4. Count beds for each day and simulation.
+
+
+### Model Parameters
+
```

Comment:
Could add all parameters from user input (i.e. admission date etc)

Reply:
Could be good to describe why certain defaults are used / ranges

Reply:
I agree, this would be great.

Reply:
Here are some notes I made when looking up doubling times

Chinese provinces
"From January 20 through February 9, the harmonic mean doubling time estimated from cumulative incidence ranged from 1.4 (95% CI, 1.2, 2.0) days (Hunan) to 3.0 (95% CI, 2.0, 4.9) days48 (Xinjiang). In Hubei, it was estimated as 2.5 (95% CI, 2.4, 2.7) days."
https://www.medrxiv.org/content/10.1101/2020.02.05.20020750v4.full.pdf

Princess diamond cruise ship
“The epidemic doubling time is at 4.6 days (95%CI: 3.0−9.3),”
https://www.medrxiv.org/content/medrxiv/early/2020/02/29/2020.02.26.20028449.full.pdf

“The epidemic doubling time (the time it takes for daily incidence to double) was 5.2 (4.6–6.1) days before Wuhan was quarantined and public health interventions implemented “
https://www.nature.com/articles/s41591-020-0822-7

Wuhan (Li et al, 2020)
In its early stages, the epidemic doubled in size every 7.4 days
https://www.nejm.org/doi/full/10.1056/NEJMoa2001316

https://link.springer.com/content/pdf/10.1007/s15010-020-01401-y.pdf
“of confirmed infections after 16th January 2020 grows at an exponential rate, with a doubling time of 1.8 days. This result is significantly different from the doubling time of 7.4 days found in Li’s group, which was obtained from analyzing the cases before 4th January 2020. This might be due to several factors: at early stages, the lack of awareness for the disease and lack of detection methods would have hindered the disease detection and confirmation. “

Lots of estimates by country
https://arxiv.org/ftp/arxiv/papers/2003/2003.08661.pdf

Reply:
@erees This is great! Can you add this as a separate PR after this one is through?